x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.tsx

```diff
@@ -224,6 +226,36 @@ CspPolicyTemplateForm.displayName = 'CspPolicyTemplateForm';
 // eslint-disable-next-line import/no-default-export
 export { CspPolicyTemplateForm as default };
 
+const usePolicyTemplateInitialName = ({
+  isEditPage,
+  isLoading,
+  integration,
+  newPolicy,
+  updatePolicy,
+}: {
+  isEditPage: boolean;
+  isLoading: boolean;
+  integration: CloudSecurityPolicyTemplate | undefined;
+  newPolicy: NewPackagePolicy;
+  updatePolicy: (policy: NewPackagePolicy) => void;
+}) => {
+  useEffect(() => {
+    if (!integration) return;
+    if (isEditPage) return;
+    if (isLoading) return;
+    const sequenceNumber = newPolicy.name.replace(/\D/g, '');
+    const currentIntegrationName = `${integration}${sequenceNumber ? `-${sequenceNumber}` : ''}`;
```

Comment:
Nit: This ternary expression in template literals is hard to read. Could we store the ternary expression in a variable? ``` js
const sequenceSuffix = sequenceNumber ? `-${sequenceNumber}` : '';
const currentIntegrationName = `${integration}${sequenceSuffix}`
```